### DISCUSSION TITLE
[request] Openhab integration / plugin

### DISCUSSION THREAD

**Discussion by nodiaque** [koush/scrypted#874] [Q&A]
Hello,

I was wondering if an openhab integration could be made? It's like HomeAssistant, just a different product.

Thank you!

**Comment by jamesfawcett** [comment]
I would love this too. I currenty use the ipcamera binding in openhab to trigger alerts and output a feed, but it would be great to use solely scrypted but somehow pass events/triggers to openhab.

**Comment by koush** [comment]
no plans to support openhab. use the rebroadcast streams if you want rtsp from scrypted.
